components/proxy/src/main/java/com/hotels/styx/admin/handlers/MetricsHandler.java

```diff
@@ -23,6 +23,7 @@
 import com.fasterxml.jackson.databind.ObjectWriter;
```

Comment:
Does this not need any additions to our end user documentation?

Reply:
I will add some.